src/Aspire.Hosting.Azure.CognitiveServices/AzureOpenAIConnectionResource.cs

```diff
@@ -0,0 +1,36 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+namespace Aspire.Hosting.ApplicationModel;
+
+/// <summary>
+/// Represents an Azure OpenAI resource.
+/// </summary>
+public class AzureOpenAIConnectionResource(ParameterResource innerResource, string? environmentVariableName) : IHasDeploymentResource
```

Comment:
Resource to represent an open ai resource as a connection string only.

Reply:
Why do we need this? Why can't we just use the AzureOpenAIResource?

Reply:
I wanted a way to configure an existing resource without provisioning. Similar to https://github.com/dotnet/aspire/issues/2395.

Reply:
Removed, out of scope